### ISSUE TITLE
Unable to render online image using XAML Image control

### ISSUE THREAD

**Issue by medhatelmasry** [dotnet/maui#6634]
### Description

I wrote this article last December 2021 whereby I was demonstrating how to render online images in a MAUI application. When I do the same  thing today, the image does not render in Android. However, it woks OK on a Windows emulator.

Refer to this blog aticle: 

https://blog.medhat.ca/2021/12/exploring-net-maui-apps-with-net-60-and.html

NOTE: I  am able to render images in Androiod using WebView as shown below:

<WebView Grid.RowSpan="2"
  Source="{Binding PictureUrl}" 
  WidthRequest="60" />

However,it looks really bad when I run the same app on Windows.

### Steps to Reproduce

Just go through this article on Visual Studio 2022 version Version 17.2.0 Preview 5.0

https://blog.medhat.ca/2021/12/exploring-net-maui-apps-with-net-60-and.html

### Version with bug

Release Candidate 2 (current)

### Last version that worked well

Preview 12

### Affected platforms

Android, Windows

### Affected platform versions

Windows 21H2 (OS Build 22000.652)

### Did you find any workaround?

Other than using WebView, I do not have a workaround.

**Comment by emorell96** [comment]
This is fixed in RC3: https://github.com/dotnet/maui/issues/6067

**Comment by jfversluis** [comment]
Yep this is a known issue with the current version unfortunately and should be fixed in the upcoming version. Thanks!

**Comment by icarrenomercado** [comment]
In case it helps , here is a workaround until the fix is released:

    ImageHandler.Mapper.AppendToMapping(nameof(ImageView.Drawable), async (handler, view) =>
    {
        if (view.Source is UriImageSource uriImageSource)
            try
            {
                byte[] imageData;
                using (var response = await _httpClient.GetAsync(uriImageSource.Uri))
                {
                    imageData = await response.Content.ReadAsByteArrayAsync();
                }
                handler.PlatformView.SetImageDrawable(new BitmapDrawable(BitmapFactory.DecodeByteArray(imageData, 0, imageData.Length)));
            }
            catch
            {
            }
        ;
    });
